src/components/Layout/Layout.stories.tsx

```diff
@@ -0,0 +1,202 @@
+import { Meta } from "@storybook/react";
+import { PropsWithChildren, ReactNode, useState } from "react";
+import { IconButton } from "src/components/IconButton";
+import { NestedScrollProvider } from "src/components/Layout/NestedScrollLayoutContext";
+import { PreventBrowserScroll } from "src/components/Layout/PreventBrowserScroll";
+import { ScrollableContent } from "src/components/Layout/ScrollableContent";
```

Comment:
Maybe import the three of these lines 4-6 from `src/index` to highlight that they're exported (i.e. I don't think they are now :-D ).